polygeist/tools/cgeist/Lib/ValueCategory.h

```diff
@@ -26,6 +29,9 @@ class ValueCategory {
   }
 
 public:
+  // TODO: Store whether an LValue is a bitfield.
+  static constexpr bool isBitField{false};
```

Comment:
We have
```cpp
  assert(!LHSLV.isBitField && "Not handling bitfields");
```
In CGExpr.cc, but I'd say we can remove that because we're not detecting whether a value is a bitfield anyway. Removing.